client/src/App.js

```diff
@@ -8,6 +8,7 @@ import Login from './pages/Login';
 import Signup from './pages/Signup';
 import { setContext } from '@apollo/client/link/context';
 import UserList from './pages/UserList';
+import ScratchOff from './pages/ScratchOff/ScrachOff';
```

Comment:
Why is this the structure? Just have the page. If you have a component put it in the component folder